src/modules/colorPicker/ColorPickerUI/Controls/ColorFormatControl.xaml

```diff
@@ -11,7 +11,7 @@
             Margin="12,16,12,0"
             Width="348"
             Height="36"
-            CornerRadius="2"
```

Comment:
The default CornerRadius of WinUI <2.5 was 2. That's what ModernWPF is using.

Might be better to just leave it at 2 as the rest of the UI still mimics the old WinUI styles and feels out of place. If/once we move to the latest (WinUI 2.6+) style we can update this as well?

Reply:
Don't know. I think we wanna look as modern as possible?

Reply:
Yeah.. everyone of those modules should be redone with the latest sizes, colors etc.. oh well, one day :)